File: AnnotationExample/src/com/training/beans/Book.java
package com.training.beans;

import com.training.annotations.SampleBook;

@SampleBook(id=1001,name = "Head First Java", price = 350.34)
public class Book {

    private long bookId;
    private String bookName;
    private double price;

    public Book() {
    }

    public Book(long bookId, String bookName, double price) {
        this.bookId = bookId;
        this.bookName = bookName;
        this.price = price;
    }

    public long getBookId() {
        return bookId;
    }

    public void setBookId(long bookId) {
        this.bookId = bookId;
    }

    public String getBookName() {
        return bookName;
    }

    public void setBookName(String bookName) {
        this.bookName = bookName;
    }

    public double getPrice() {
        return price;
    }

    public void setPrice(double price) {
        this.price = price;
    }

    @Override
    public String toString() {
        return "Book{" +
                "bookId=" + bookId +
                ", bookName='" + bookName + '\'' +
                ", price=" + price +
                '}';
    }
}

File: TypeScriptExample/Book.ts
class Book{
    private bookNo:number;
    private bookName:string;
    private price:number;

    constructor(bookNo: number, bookName: string, price: number) {
        this.bookNo = bookNo;
        this.bookName = bookName;
        this.price = price;
    }

    toString():string{
        return `${this.bookNo},${this.bookName},${this.price}`
    }
}